remote/src/test/scala/org/apache/pekko/remote/classic/ActorsLeakSpec.scala

```diff
@@ -41,6 +41,7 @@ object ActorsLeakSpec {
        pekko.remote.classic.quarantine-after-silence = 3 s
        pekko.remote.use-unsafe-remote-features-outside-cluster = on
        pekko.test.filter-leeway = 12 s
+       pekko.test.timefactor = 2
```

Comment:
Maybe this will always be needed on some box.

Reply:
Yes, its currently set in CI so this isn't a concern. The bigger problem was that when you ran this test locally it basically never passed when you just did `sbt test`.

Reply:
Maybe you need increase the jvm xmx too？